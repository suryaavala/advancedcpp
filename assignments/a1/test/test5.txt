.1
.2 add
.3 mult
.4 div
.5 sub
.6 sqrt