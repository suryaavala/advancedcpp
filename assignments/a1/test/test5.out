0.200 + 0.100 = 0.300
0.300 * 0.300 = 0.090
0.400 / 0.090 = 4.444
0.500 - 4.444 = -3.944
sqrt 0.600 = 0.775
